vocabularies/qld-resource-permit-status.ttl

```diff
@@ -0,0 +1,214 @@
+    skos:broader gmps:application,
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Conditionally surrendered"@en .
+
+gmps:conversion-of-ml-to-mc a skos:Concept ;
+    skos:definition "The Mining Lease has been converted to a Mining Claim."@en ;
+    skos:broader gmps:granted ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Conversion of ML to MC"@en .
+
+gmps:expired a skos:Concept ;
+    skos:definition "The Resource Permit has expired."@en ;
+    skos:broader gmps:granted,
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Expired"@en .
+
+gmps:finalised a skos:Concept ;
+    skos:definition "The Resource Permit has no outstanding requirements."@en ;
+    skos:broader gmps:non-current ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Finalised"@en .
+
+gmps:in-progress a skos:Concept ;
+    skos:definition "The Application is in-progress."@en ;
+    skos:broader gmps:discarded,
+        gmps:prelodgement ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "In progress"@en .
+
+gmps:on-hold-oil-shale-application a skos:Concept ;
+    skos:definition "The oil-shale application has been placed on-hold."@en ;
+    skos:broader gmps:application ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "On-hold oil shale application"@en .
+
+gmps:paid a skos:Concept ;
+    skos:definition "Application payment has been made and confirmation of payment has been received."@en ;
+    skos:broader gmps:discarded,
+        gmps:submitted ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Paid"@en .
+
+gmps:payment-authorised a skos:Concept ;
+    skos:definition "Payment authorisation has been requested."@en ;
+    skos:broader gmps:prelodgement,
+        gmps:submitted ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Payment authorised"@en .
+
+gmps:payment-pending a skos:Concept ;
+    skos:definition "Payment is pending."@en ;
+    skos:broader gmps:discarded,
+        gmps:submitted ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Payment pending"@en .
+
+gmps:preferred-tenderer a skos:Concept ;
+    skos:definition "The applicant is the preferred tenderer."@en ;
+    skos:broader gmps:application,
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Preferred Tenderer"@en .
+
+gmps:priority-applicant a skos:Concept ;
+    skos:definition "The applicant is a Priority Applicant for competing application."@en ;
+    skos:broader gmps:application ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Priority Applicant"@en .
+
+gmps:ranked a skos:Concept ;
+    skos:definition "The application for the Resource Permit is a Ranked Application, pending grant of a Priority Application."@en ;
+    skos:broader gmps:application ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Ranked"@en .
+
+gmps:rejected a skos:Concept ;
+    skos:definition "The Resource Permit has been rejected."@en ;
```

Comment:
The permit has been rejected, or the application for a permit has been rejected?

Reply:
concept removed as the same as unsuccessful-application. marking conversation as resolved.